openedx/features/edly/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django.contrib.sites.models import Site
+from django.core.validators import RegexValidator
+
+
+class Organization(models.Model):
+    alphanumeric = RegexValidator(r'^[0-9a-zA-Z]*$', 'Only alphanumeric characters are allowed.')
+
+    name = models.CharField(max_length=100, unique=True, validators=[alphanumeric])
```

Comment:
Name can be anything remove alphanumeric validator.
Also the length of name should be 255 just like https://github.com/edx/edx-organizations/blob/master/organizations/models.py
Remove unique constraint as well

Reply:
@zubair-arbi this restriction is added to cater to acceptance criteria point 1.
`create edly organization model (edly_panel.apps.organizations.models.Organization) with name and slug (unique identifier only alphanumeric value is allowed)`

Reply:
I mentioned that alphanumeric check is for slug field only

Reply:
I mentioned that alphanumeric check is for slug field only

Reply:
I mentioned that alphanumeric check is for slug field only

Reply:
I mentioned that alphanumeric check is for slug field only

Reply:
I mentioned that alphanumeric check is for slug field only

Reply:
I mentioned that alphanumeric check is for slug field only

Reply:
I mentioned that alphanumeric check is for slug field only

Reply:
I mentioned that alphanumeric check is for slug field only

Reply:
Done